src/Microsoft.ML/Runtime/Internal/Tools/GeneratorUtils.cs

```diff
@@ -0,0 +1,467 @@
+﻿using Microsoft.CSharp;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Newtonsoft.Json.Linq;
+using System;
+using System.CodeDom;
```

Comment:
Other files in this project seem to have a `System` then `Microsoft` ordering. We should probably harmonize this somehow at some point, but for now might be nice to keep consistency at the project level.